web/e2e/home.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test';
```

Comment:
may be call the directory playwright instead of e2e? Normally ppl name their cypress test folders as cypress, what is the general convention u see in the wild for playwright? Do they call it e2e?